packages/react-instantsearch-hooks-web/src/ui/__tests__/RefinementList.test.tsx

```diff
@@ -645,4 +650,20 @@ describe('RefinementList', () => {
       'Some custom title'
     );
   });
+
+  test('renders with translations', () => {
+    const props = createProps({
+      showMore: true,
+      translations: {
+        showMoreButtonText({ isShowingMore }) {
+          return isShowingMore ? 'Show less brands' : 'Show more brands';
+        },
+      },
+    });
+    const { getByRole } = render(<RefinementList {...props} />);
+
+    expect(
+      getByRole('button', { name: 'Show more brands' })
+    ).toBeInTheDocument();
```

Comment:
Should we re-render with `showMore=false` to test the other side of the ternary?